tests/test-option-parser.sh

```diff
@@ -32,7 +32,7 @@ check_output_inner ()
     func_quote pretty ${1+"$@"}
     $ECHO "[[ output check ]] args: $func_quote_result"
 
-    output=`$helper ${1+"$@"} 2>/dev/null`
+    output=`$__GL_ALL_SHELLS_SHELL "$helper" ${1+"$@"} 2>/dev/null`
```

Comment:
Sorry, forgot to finish.  Other solution is to define a quick shell var,
eg:

hshell=$__GL_ALL_SHELLS_SHELL 

output=`$hshell "$helper" ${1+"$@"} 2>/dev/null` 

... which is pretty clean looking :) 

Scott 

On 2016-08-17 14:38, Pavel Raiskup wrote:

> In tests/test-option-parser.sh [1]:
> 
>> @@ -32,7 +32,7 @@ check_output_inner ()
>> func_quote pretty ${1+"$@"}
>> $ECHO "[[ output check ]] args: $func_quote_result"
>> 
>> -    output=`$helper ${1+"$@"} 2>/dev/null`
>> +    output=`$__GL_ALL_SHELLS_SHELL "$helper" ${1+"$@"} 2>/dev/null`
> 
> Thanks for your work! 
> 
> Maybe it is detail, but could we have rather $helper properly quoted and later eval()ed to not copy the ugly $__GL_ALL_SHELLS_SHELL on multiple places? 
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [2], or mute the thread [3].
 

Links:
------
[1]
https://github.com/gnulib-modules/bootstrap/pull/7#discussion_r75113724
[2]
https://github.com/gnulib-modules/bootstrap/pull/7/files/20a6d66266904c00c29f943b71c4f8424db1b4d5#r75113724
[3]
https://github.com/notifications/unsubscribe-auth/AAnGuNF8tF3x--QsugNT8rGQcLxcp_JJks5qgwCygaJpZM4JlzUb